sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -581,6 +581,38 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
     }
   }
 
+  test("insert overwrite to not exists dir from hive metastore table") {
```

Comment:
What's the difference from the `"insert overwrite to not exist local dir"` test?

Reply:
One is used for test result correct or not, another is used for test the not exist path created or not.

Reply:
Can you run these tests in a single case?

Reply:
Yes,I can remove "insert overwrite to not exist local dir" test and retain another.